docs/content/v0.20/examples.md

```diff
@@ -17,11 +17,13 @@ The following steps establish an initial state used in the examples below.
    kflex init --create-kind
    ```
 
-2. Update the post-create-hooks in KubeFlex to install kubestellar with the v0.20.0-alpha.1 images:
+  If you are installing KubeStellar on an existing OpenShift cluster, just use the command `kflex init`.
```

Comment:
This needs to be indented three spaces in order for markdown to handle it properly.
The attached screen snapshot shows what I currently see, which is this line not visually nested under "1.".

<img width="65" alt="Screenshot 2024-02-21 at 11 16 49 PM" src="https://github.com/kubestellar/kubestellar/assets/14296719/d23a31c6-2f7e-43fc-9fb7-13e1d74098a5">

Reply:
ok, done